.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "subprojects/wf-config"]
 	path = subprojects/wf-config
 	url = https://github.com/ammen99/wf-config
+[submodule "subprojects/wlroots"]
+	path = subprojects/wlroots
+	url = https://github.com/swaywm/wlroots
```

Comment:
Do we want to use the upstream url or one under our control? If we use a url that is within our control, we can update wlroots and fix up wayfire at our leisure and this should make it less likely for users to experience breakage.

Reply:
I don't think we will have many patches for wlroots, and it generally makes sense to keep as close to upstream as possible. Of course, we can adjust the submodule to any commit we want, so I see no problems to change the submodule to our own fork if the need arises.

Reply:
I would like to see https://github.com/swaywm/sway/issues/1865#issuecomment-450659415 for one.

Reply:
alright, I'll fork wlroots

Reply:
you can send it as a PR to WayfireWM/wlroots